### ISSUE TITLE
multiples s3 for backup

### ISSUE THREAD

**Issue by sherlant** [vmware-tanzu/velero#6744]
Hi,
i want to find a way to use multiple s3 to backup my data. With Velero, there is a way to do this ?
Can i create a new instance of velero in my cluster, with corrects credentials for the second s3, or it is possible from an another way ?

Thx

**Comment by mateusoliveira43** [comment]
Yes, you can use multiple Backup Storage Locations (S3 buckets) for backups, no need for a second instance of velero. Check documentation about it https://velero.io/docs/v1.12/locations/

**Comment by sherlant** [comment]
Ok, i have not see the creds section in bsl to allow multiples s3. Thanks
